src/core/datatypes/trees/Tree.cpp

```diff
@@ -950,19 +951,40 @@ void RevBayesCore::Tree::checkTaxonAges(bool forceAdjust)
 {
     for (auto& node : nodes)
     {
-        if(!node->isTip()) continue;
+        if ( !node->isTip() ) continue;
         Taxon taxon = node->getTaxon();
-        if(node->getAge() < taxon.getMinAge()) {
-            if(forceAdjust) {
+        
+        if ( node->getAge() < taxon.getMinAge() )
+        {
+            double diff = taxon.getMinAge() - node->getAge();
+            if (forceAdjust)
+            {
                 node->setAge(taxon.getMinAge());
-                RBOUT("Age of taxon " + taxon.getName() + " was below the specified minimum and has been adjusted.");
-            } else throw RbException() << "Age of taxon " << taxon.getName() << " is below the specified minimum.";
+                
+                // only notify the user if we are making a non-trivial adjustment
+                if ( diff > RbSettings::userSettings().getTolerance() )
+                    RBOUT("Age of taxon " + taxon.getName() + " was below the specified minimum and has been adjusted.");
+            }
+            else
+            {
+                throw RbException() << "Age of taxon " << taxon.getName() << " is below the specified minimum.";
+            }
         }
-        if(node->getAge() > taxon.getMaxAge()) {
-            if(forceAdjust) {
+        if ( node->getAge() > taxon.getMaxAge() )
+        {
+            double diff = node->getAge() - taxon.getMaxAge();
+            if (forceAdjust)
+            {
                 node->setAge(taxon.getMaxAge());
-                RBOUT("Age of taxon " + taxon.getName() + " was above the specified maximum and has been adjusted.");
-            } else throw RbException() << "Age of taxon " << taxon.getName() << " is above the specified maximum.";
+                
+                // only notify the user if we are making a non-trivial adjustment
+                if ( diff > RbSettings::userSettings().getTolerance() )
+                    RBOUT("Age of taxon " + taxon.getName() + " was above the specified maximum and has been adjusted.");
```

Comment:
Possibly we should say adjusted from old_age -> new_age?

Reply:
Good idea – done.

Reply:
In theory you could say how much the age was adjusted by, instead of adjusted from x -> y.  But you could also just merge it now.